pinot-hadoop/pom.xml

```diff
@@ -43,13 +43,24 @@
         <plugins>
           <plugin>
             <artifactId>maven-shade-plugin</artifactId>
+            <version>3.2.1</version>
             <executions>
               <execution>
                 <phase>package</phase>
                 <goals>
                   <goal>shade</goal>
                 </goals>
                 <configuration>
+                  <relocations>
```

Comment:
Why shading these jars?

Reply:
This is to relocate method call to use our shaded jar not from hadoop classpath. For Hadoop env, there could be older version of jackson and guava in classpath and cause NoSuchMethod exception.

Reply:
Can you please add some comments to the commit message?